javascript-sdk/examples/close-cookie-dialog/close-cookie-dialog.js

```diff
@@ -0,0 +1,38 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+
+require('dotenv').config();
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  configure({ apiKey: process.env.AGENTQL_API_KEY });
```

Comment:
Recommend removing this and covering the API key in the installation instructions